### DISCUSSION TITLE
404 page after Explorer is published

### DISCUSSION THREAD

**Discussion by FrancescoBini** [WalletConnect/walletconnect-monorepo#3565] [Developer Support] [answered]
Our wallet has been approved for the WalletConnect Explorer as seen [here](https://walletconnect.com/explorer?search=kresus) but when clicked it returns a 404 error page as shown [here](https://walletconnect.com/explorer/kresus-superapp). 

Could anyone please help? 

Thank you

**Comment by glitch-txs** [comment]
Looking into it, thanks for reporting 🤝

**Reply by FrancescoBini** [reply]
Hi, thank you for looking into it. Do you have updates?

**Reply by boidushya** [reply]
Hey @FrancescoBini , our team's taking a look at it and we should roll out a fix soon

**Comment by PROrector** [comment]
Similar [problem](https://walletconnect.com/explorer/trustee-wallet). I suspect it's because of the hyphen between the words of the title.

**Reply by boidushya** [reply]
cc. @Cali93, if we can repro this, lets take a look at this issue

**Reply by Cali93** [reply, marked as answer]
Thanks for reporting the issue. There was an issue related to the build of the Explorer page.
It has been fixed and you should be able to see your project if it has been approved.
